### PR TITLE
fluxbox lightweight window manager

### PR DESCRIPTION
**Pull request by luigi-calori**
This is a lightweight minimal window manager to be used in vnc sessions on headless clusters

### CODE REVIEW

File: var/spack/repos/builtin/packages/fluxbox/package.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    homepage = "http://fluxbox.org/"
+    url      = "http://sourceforge.net/projects/fluxbox/files/fluxbox/1.3.7/fluxbox-1.3.7.tar.gz"
+
+    version('1.3.7', 'd99d7710f9daf793e0246dae5304b595')
+
+    # depends_on('m4', type='build')
+    # depends_on('autoconf', type='build')
+    # depends_on('automake', type='build')
+    # depends_on('libtool', type='build')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/fluxbox/package.py]: Can you either uncomment or remove commented dependencies?

Comment (luigi-calori) [reply var/spack/repos/builtin/packages/fluxbox/package.py]: Sorry, I ' ve left them as I do not know which is the right policy:

I' ve tested it with dependency commented on all of our cluster deployment.

I' ve tried a build on a docker ubuntu and it seem to work.

Nevertheless I do not know which is the correct policy:
are automake and autoconf dependencies already ensured by deriving from AutotoolsPackage ?
Which is the suggested policy regarding base build tools like libtool, binutils, m4 ?

For me the easiest path is to remove all commented depends.

I' m willing to try one of the docker based approach for testing builds but not there yet.

Comment (alalazo) [reply var/spack/repos/builtin/packages/fluxbox/package.py]: You need these dependencies only for packages that don't ship with a `configure` script. For instance if you have a `develop` version tied to a repository checkout.

**Comment by alalazo** [comment]
@luigi-calori As you permitted edits from maintainers, I took the freedom of doing a minor modification myself instead of iterating over the indentation of docstrings. Let me know if you'd like to proceed otherwise in the future.

I'll merge this as soon as Travis finishes.

**Comment by luigi-calori** [comment]
@alalazo Thanks a lot and  edit more than welcome.
I was not aware of  the maintainer edit feature.
For forthcoming PR, use it freely.
On my side I' ll try to learn by example the proper code style :-)

**Pull request merged**